DigitalLearningSolutions.Web/Scripts/trackingSystem/emailDelegates.ts

```diff
@@ -0,0 +1,26 @@
+import { SearchSortFilterAndPaginate } from '../searchSortFilterAndPaginate/searchSortFilterAndPaginate';
+
+const selectedElements = document.querySelectorAll('.delegate-checkbox:checked') as NodeListOf<HTMLInputElement>;
+const selectedIds = Array.from(selectedElements).map((el) => el.value);
+const queryString = '?selectedIds='.concat(selectedIds.join('&selectedIds='));
+const route = `TrackingSystem/Delegates/Email/AllEmailDelegateItems${queryString}`;
+
+// eslint-disable-next-line no-new
+new SearchSortFilterAndPaginate(route, true, 'EmailDelegateFilter', false, false, false);
+
+const selectAll = () => {

```

Comment:
It's technically fine, but is there any particular reason we're declaring these in a different style to the other typescript stuff?